src/sdk/wallet.spec.js

```diff
@@ -0,0 +1,178 @@
+'use strict';
```

Comment:
This spec file looks a bit thin compared to the production code it's testing :-P

Reply:
A bit, but depositing isn't that complex really. Its testing all the major happy paths, but is lacking in the failure scenario department :)